app/shared/db/breakpoints.server.ts

```diff
@@ -28,12 +31,29 @@ export const create = async (
   treeId: Project["id"],
   breakpoints: Array<Breakpoint>
 ) => {
-  const data = {
+  await prisma.breakpoints.create({
+    data: {
+      treeId,
+    },
+  });
+
+  const all = breakpoints.map(async (breakpoint: Breakpoint) => {
+    new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
Without this it breaks on a timeout

From my digging the database locks when you make one request

Reply:
makes sense since sqlite is runs with no network between, it expects each query come sequentially, lets try using https://www.prisma.io/docs/concepts/components/prisma-client/transactions

Reply:
this just worked, never used them but yeah it worked perfectly